docs/INSTALL.md

```diff
@@ -19,6 +19,7 @@ Copyright 2019 Intel Corporation
 
 You need the following dependencies.
 
+* Atleast 16-17 CPU are required for deploying BESS successfully on your system. So, allocate around ```18 cores/vCPU```.
```

Comment:
Why?

Reply:
Because When I was trying to deploy it on my VM it was giving an error saying: you need at least 15-16 cores to deploy it. That why I gave 18 core and try to deploy it and it worked.

Reply:
Well, not really... I think you got that error message because the docker_setup.sh script requests to run the container for BESS in cpus 12 and 13 (as you can see [here](https://github.com/omec-project/upf/blob/master/scripts/docker_setup.sh#L155)).
```
# Run bessd
docker run --name bess -td --restart unless-stopped \
	--cpuset-cpus=12-13 \
	--ulimit memlock=-1 -v /dev/hugepages:/dev/hugepages \
```

 By accordingly adjusting the assigned cpus to BESS you can avoid this issue. For example, I just ran the UPF in a VM that has 8 cores by properly adjusting the `--cpuset-cpus` values
```diff
diff --git a/scripts/docker_setup.sh b/scripts/docker_setup.sh
index f5946c1..88f19a2 100755
--- a/scripts/docker_setup.sh
+++ b/scripts/docker_setup.sh
@@ -152,7 +152,7 @@ fi
 
 # Run bessd
 docker run --name bess -td --restart unless-stopped \
-       --cpuset-cpus=12-13 \
+       --cpuset-cpus=4-6 \
        --ulimit memlock=-1 -v /dev/hugepages:/dev/hugepages \
        -v "$PWD/conf":/opt/bess/bessctl/conf \
        --net container:pause \
```

Without adjusting the above, you will get an error message like this one:
```
docker: Error response from daemon: Requested CPUs are not available - requested 12-13, available: 0-7.
```

Reply:
Okay Got it. Thanks for sharing.And 
How much minimum CPUs are required for deployment of BESS?

Reply:
It depends on the performance you want to get out of BESS, but the minimum should be 1. As I mentioned in my previous comment, `cpuset-cpus=12-13` are assigned by default, which basically indicates that 2 cores are assigned.

Reply:
Okay, thanks for responding.